packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -54,6 +55,16 @@ export class Cluster extends cdk.Construct implements ICluster {
    */
   public readonly clusterName: string;
 
+  /**
+   * The name of the service discovery namespace created in this cluster
+   */
+  public serviceDiscoveryNamespace: string;
```

Comment:
Actually I think I'd like to replace both of these members with a single `INamespace` object, right?